types/htmlhint/reporter.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Hint, Rule, Ruleset } from './types';
+
+export default class Reporter {
+    html: string;
+    lines: string[];
+    brLen: number;
+    ruleset: Ruleset;
+    messages: Hint[];
+
+    constructor(html: string, ruleset: Ruleset);
+    info(message: string, line: number, col: number, rule: Rule, raw: string): void;
+    warn(message: string, line: number, col: number, rule: Rule, raw: string): void;
+    error(message: string, line: number, col: number, rule: Rule, raw: string): void;
```

Comment:
```suggestion
    error(message: string, line: number, col: number, rule: Rule, raw: string): void;
     private report;
```
just to align wiht original (I understand it cannot be consumed, but if someone regenerates, it would create a diff)